Source/Tools/CMakeLists.txt

```diff
@@ -3,6 +3,14 @@ add_subdirectory(CommonTools)
 if (NOT MINGW_BUILD)
   if (BUILD_FEXCONFIG)
     add_subdirectory(FEXConfig/)
```

Comment:
Can we have a conditional so that we can only build this variant and not the other?

Reply:
Sounds reasonable. `BUILD_FEXCONFIG` is now called `USE_FEXCONFIG_TOOLKIT` and can be set to either ""/"qt"/"imgui". Since it's an exclusive choice, FEXQonfig has been renamed to FEXConfig.